spec/traits.dd

```diff
@@ -1203,6 +1203,7 @@ version (CppRuntime_Microsoft)
 
         $(UL
         $(LI $(D "cppRuntimeLibrary") - The C++ runtime library affinity for this toolchain)
+        $(LI $(D "cppStd") - The version of C++ standard supported in $(D extern$(LPAREN)C++$(RPAREN) code)
```

Comment:
Specifically it is the numerical value of `__cplusplus`, please make that clear (e.g. with an example or text that says that)

Reply:
Added `equivalent to the __cplusplus macro in a C++ compiler`.